Resources/Prototypes/Entities/Mobs/Species/moth.yml

```diff
@@ -127,3 +129,5 @@
   components:
   - type: HumanoidAppearance
     species: Moth
+  - type: Inventory
+    speciesId: moth
```

Comment:
Is this a duplicate?

Reply:
For the same reason as the arachnid one, it's so the moth sprites show up in the pre-round lobby screen because they wouldn't show up beforehand.